packages/p2p/src/components/modal-manager/modals/quick-add-modal/quick-add-modal-content/choose-payment-methods/choose-payment-methods.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { Localize } from 'Components/i18next';
+import BuyAdPaymentMethodsList from 'Pages/my-ads/buy-ad-payment-methods-list.jsx';
+
+type TChoosePaymentMethodsProps = {
+    is_payment_methods_selected: boolean;
+    selected_methods: string[];
+    setSelectedMethods: React.Dispatch<React.SetStateAction<string[]>>;
+};
+
+const ChoosePaymentMethods = ({
+    is_payment_methods_selected,
+    selected_methods,
+    setSelectedMethods,
+}: TChoosePaymentMethodsProps) => {
+    return (
+        <React.Fragment>
+            <div className='quick-add-modal-content__info'>
+                <Text color='prominent' size={isMobile() ? 'xxs' : 'xs'}>
```

Comment:
i think we need to change this to is_mobile

Reply:
sure can, yeah better to start replacing any instances of isMobile with is_mobile from now on